spread-tests/cgroup-used/task.yaml

```diff
@@ -0,0 +1,36 @@
+summary: Check that launcher cgroup functionality works
+# This is blacklisted on debian because we first have to get the dpkg-vendor patches
+systems: [-debian-8]
+restore: |
+    snap remove hello-world
+    rm -f /etc/udev/rules.d/70-spread-test.rules
+    udevadm control --reload-rules
+    udevadm settle
+    udevadm trigger
+    udevadm settle
+    # no way to clear cgroups atm
+execute: |
+    cd /
+    udevadm settle
+    echo Install hello-world
+    snap install hello-world
+    echo Clear udev tags and cgroups with non-test device and running hello-world
+    echo 'KERNEL=="uinput", TAG+="snap_hello-world_echo"' > /etc/udev/rules.d/70-spread-test.rules
+    udevadm control --reload-rules
+    udevadm settle
+    udevadm trigger
+    hello-world.echo | grep Hello
+    echo Verify no tags for hello-world.echo for kmsg
+    if udevadm info /sys/devices/virtual/mem/kmsg | grep snap_hello-world_echo ; then exit 1; fi
+    echo Manually add udev tags for hello-world.echo for kmsg
+    echo 'KERNEL=="kmsg", TAG+="snap_hello-world_echo"' > /etc/udev/rules.d/70-spread-test.rules
+    echo Simulate snapd udev triggers
+    udevadm control --reload-rules
```

Comment:
Not 100% sure but udev tracks this nowadays so I think those reload bits are not strictly mandatory. Let's leave them but just saying.